package.json

```diff
@@ -70,15 +70,20 @@
     "@commitlint/config-conventional": "^17.8.1",
     "@typescript-eslint/eslint-plugin": "^6.21.0",
     "@typescript-eslint/parser": "^6.21.0",
-    "cspell": "^8.8.4",
+    "bson": "^6.2.0",
+    "chance": "^1.1.11",
+    "cookie-signature": "^1.2.1",
+    "cspell": "^8.8.1",
     "eslint": "^8.57.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-prettier": "^5.1.3",
     "husky": "^8.0.3",
+    "jws": "^4.0.0",
```

Comment:
we seem to have this and `jsonwebtoken` -- while this is only a dev dep it might be worth to try and remove this